analyzer/codechecker_analyzer/analyzers/cppcheck/analyzer.py

```diff
@@ -83,6 +83,29 @@ def get_analyzer_mentioned_files(self, output):
         """
         pass
 
+    def parse_analyzer_config(self):
+        "Parses a white listed compiler flags"
+        params = []
+        interesting_option = re.compile("^-[I|U|D].*$")
```

Comment:
```suggestion
        interesting_option = re.compile("-[I|U|D]")
```
Maybe it's just a style issue, so feel free to ignore it, but this is supposed to be the same. `re.match()` finds the pattern at the beginning of the string by default.

Reply:
I think the part ".*" that matches to any sequence is needed, before the dollar sign, because on line 98 the matched span length is used to determine if it is paramater that is defined with a space. A span of (0, 2) means it is -I /abc/ and -I/abc/ will be matched as a span of (0, 5). Maybe the dot wildcard without the kleene star is enough. I'll check.

Reply:
Oh, you're right :)